pensieve/analysis.py

```diff
@@ -134,7 +134,7 @@ def _publish_view(self, window_period: AnalysisPeriod, table_prefix=None):
             )
             """
         )
-        self.bigquery.execute(sql)
+        _ = self.bigquery.execute(sql)
```

Comment:
This is odd. It does seem like the queries get executed but unless the result is assigned to a variable, no data is actually written to BigQuery...

Reply:
More or less debugging by trail and error. I was executing `pensieve run` locally and based on log messages it seemed like the query is executed in BigQuery, however when looking for the table in the console it wasn't there. This change fixed that.

Reply:
I think this is too spooky to accept -- these calls should be generating identical bytecode but for an additional assignment operation and the result should exist whether we hang on to it or not. Like I think are options are a) this didn't fix it and the real problem still exists, b) we've exposed some serious problem (maybe some race related to object lifetimes?), or c) this is haunted and we should call an exorcist before taking up quiet lives farming potatoes in Pemberton. I think a is most likely.

Reply:
🕵️‍♀️ Yeah, I'll keep investigating